<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterPartOfWindow</class>
 <widget class="QMainWindow" name="FilterPartOfWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_editing_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>98</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>By Part Of:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>(*) Required fields</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>241</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lbl_entity">
         <property name="text">
          <string>Entity *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Predefined type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_relation">
         <property name="text">
          <string>Relation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="CustomLineEdit" name="txt_entity"/>
       </item>
       <item>
        <widget class="CustomLineEdit" name="txt_predef_type"/>
       </item>
       <item>
        <widget class="CustomLineEdit" name="txt_relation"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="combo_optionality">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>required</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>prohibited</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_optionality">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Optionality</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header>myWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
